Makefile

```diff
@@ -151,6 +151,9 @@ MANIFEST_ROOT ?= config
 CRD_ROOT ?= $(MANIFEST_ROOT)/crd/bases
 WEBHOOK_ROOT ?= $(MANIFEST_ROOT)/webhook
 RBAC_ROOT ?= $(MANIFEST_ROOT)/rbac
+ASO_CRDS_PATH := $(MANIFEST_ROOT)/aso/crds.yaml
+ASO_VERSION := v2.0.0
```

Comment:
Not super thrilled that the ASO version now needs to be updated here, in config/aso/kustomization.yaml, and go.mod. Any hints to consolidate those at all are appreciated!

This might be too gnarly, but:
```
ASO_VERSION := $(shell go list -m -f '{{.Version}}' github.com/Azure/azure-service-operator/v2)
```

Reply:
If you can solve that problem (needing to do dual-bookkeeping in `Makefile` and in `go.mod` for some packages), we could apply it to other things like Ginkgo and `tilt`.